Update Podsights destination support email [SECOPS-15401] (#7711)
segmentio/segment-docs: src/connections/destinations/catalog/podsights/index.md

});
```

> Track calls will be mapped to Podsights events. Podsights supports the following from the Segment Spec:

Reviewer: ```suggestion
Track calls are mapped to Podsights events. Podsights supports the following from the Segment Spec:
```
